src/main/java/mate/academy/hibernate/relations/exception/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.hibernate.relations.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message) {
```

Comment:
```suggestion
    public DataProcessingException(String message, Throwable cause) {
```
let's also pass the actual root exception to the constructor